src/modules/returnToNCI/returnToNCI-bar.js

```diff
@@ -255,15 +257,24 @@ var merge = function() {
             // var barStyles = '<link rel="stylesheet" href="/modules/returnToNCI/returnToNCI-bar.css" />';
 
 
+            var target = "_parent";
+
+            if (settings.uxWindow !== false) {
+                target = "_blank";
```

Comment:
The settings.uxWindow defaults to being false.  If the value of uxWindow is false, then the target of the menu links shall be _parent.  In the event the uxWindow is set to something other than false, the target of the menu links shall be the value of uxWindow.   (The configurer may choose "_blank" or a named window.

Reply:
Ok, I'll make the change and then retest.